resources/Microsoft.VSCode.Dsc/Microsoft.VSCode.Dsc.psm1

```diff
@@ -95,8 +95,8 @@ function Invoke-VSCode {
         [string]$Command
     )
 
-    $stdErrTempFile = "$env:TEMP\$((New-Guid).Guid)"
-    $stdOutTempFile = "$env:TEMP\$((New-Guid).Guid)"
+    $stdErrTempFile = [System.IO.Path]::GetTempPath() + (New-Guid).Guid
+    $stdOutTempFile = [System.IO.Path]::GetTempPath() + (New-Guid).Guid
```

Comment:
Now that I think about it, string concatenation for path is probably not the way to go. Can you change this to use `Join-Path`?

Reply:
Nitter. Just updated it.